edk2toolext/environment/uefi_build.py

```diff
@@ -30,6 +30,15 @@
 import datetime
 
 
+class PlatformEnv():
+    """Container representing a Platform Environment Variable."""
+    def __init__(self, name, description, default):
```

Comment:
please add typing

Reply:
PlatformEnv class removed in preference of a namedtuple